### PR TITLE
Triple fusion with RGB-D cameras and data collection with it

### PR DESCRIPTION
**Pull request by YutoUchimi**
Close https://github.com/start-jsk/jsk_apc/issues/2029

Test phase for introducing stereo Astra mini S camera into baxter's hands.
For now, the stereo camera is set only on his right hand.

Usage:
------
- Pick task
```
$ roslaunch jsk_arc2017_baxter baxter.launch stereo:=true pick:=true
```
- Stow task
```
$ roslaunch jsk_arc2017_baxter baxter.launch stereo:=true stow:=true
```
- Collect data
```
$ roslaunch jsk_arc2017_baxter baxter.launch stereo:=true collect_data:=true
$ roslaunch jsk_arc2017_baxter collect_data_in_shelf.launch
```

### CODE REVIEW

**Comment by wkentaro** [comment]
image

**Comment by wkentaro** [comment]
## TODO

- [ ] Multiple view (9 or 12 views)
  - Save which view is selected for current sava

**Comment by YutoUchimi** [comment]
I also collected multi-view data with Astra Mini S.
See https://github.com/start-jsk/jsk_apc/issues/2029#issuecomment-308424205

**Comment by YutoUchimi** [comment]
@wkentaro  Please review this.

**Comment by knorth55** [comment]
How about 
`right_hand_right_camera` -> `right_hand_stereo/right_camera`?
`right_hand_left_camera` -> `right_hand_stereo/left_camera`?

File: jsk_arc2017_baxter/launch/setup/baxter.launch [hunk 1]
```diff
@@ -1,7 +1,10 @@
 <launch>
 
   <arg name="moveit" default="true" />
-  <arg name="pick" default="true" />
+  <arg name="pick" default="false" />
+  <arg name="stow" default="false" />
+  <arg name="collect_data" default="false" />
```
Comment (wkentaro) [inline jsk_arc2017_baxter/launch/setup/baxter.launch]: How about `<arg name="task" default="pick" />`

```
<rosparam command="load" if="$(arg stow)" file="$(find jsk_arc2017_common)/config/$(arg task)/shelf_marker.yaml"/>
```

**Comment by wkentaro** [comment]
> How about
right_hand_right_camera -> right_hand_stereo/right_camera?
right_hand_left_camera -> right_hand_stereo/left_camera?

Seeing multisense publishing topic, `xxx/left/xxx` and `xxx/right/xxx` seem common.
I propose below topic names. What do you think? @knorth55 @YutoUchimi 

- `right_hand_stereo/left/rgb/camera_info`
- `right_hand_stereo/left/rgb/image_rect_color`
- `right_hand_stereo/left/depth_registered/points`
- `right_hand_stereo/left/depth_registered/hw_registered/image_raw_rect`
- `right_hand_stereo/right/rgb/camera_info`
- `right_hand_stereo/right/rgb/image_rect_color`
- `right_hand_stereo/right/depth_registered/hw_registered/image_raw_rect`
- `right_hand_stereo/right/depth_registered/points`
- `right_hand_stereo/fused/rgb/camera_info`
- `right_hand_stereo/fused/rgb/image_rect_color`
- `right_hand_stereo/fused/depth_registered/hw_registered/image_raw_rect`
- `right_hand_stereo/fused/depth_registered/points`

**Comment by knorth55** [comment]
>Seeing multisense publishing topic, xxx/left/xxx and xxx/right/xxx seem common.
I propose below topic names. What do you think? @knorth55 @YutoUchimi 
    right_hand_stereo/left/rgb/camera_info
    right_hand_stereo/left/rgb/image_rect_color
    right_hand_stereo/left/depth_registered/points
    right_hand_stereo/left/depth_registered/hw_registered/image_raw_rect
    right_hand_stereo/right/rgb/camera_info
    right_hand_stereo/right/rgb/image_rect_color
    right_hand_stereo/right/depth_registered/hw_registered/image_raw_rect
    right_hand_stereo/right/depth_registered/points
    right_hand_stereo/fused/rgb/camera_info
    right_hand_stereo/fused/rgb/image_rect_color
    right_hand_stereo/fused/depth_registered/hw_registered/image_raw_rect
    right_hand_stereo/fused/depth_registered/points

+1

**Comment by knorth55** [comment]
@YutoUchimi can you update this PR?

**Pull request merged**